pkgs/by-name/pa/paperlib/package.nix

```diff
@@ -0,0 +1,75 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  appimageTools,
+  undmg,
+}:
+let
+  pname = "paperlib-bin";
```

Comment:
I'm very sorry for not elaborating it enough.

By renaming the package to paperlib-bin, I mean moving `pkgs/by-name/pa/paperlib` to `pkg/by-name/pa/paperlib-bin`, while keeping `pname = "paperlib";` unchanged.